Dockerfile

```diff
@@ -2,24 +2,16 @@
 
 
 # Build image 
-FROM node:13.5.0-alpine3.11
+FROM node:14-alpine
 
 WORKDIR /app
 
 # Copy in dependencies for building
-COPY *.json ./
-COPY yarn.lock ./
-COPY config ./config
-COPY integrations ./integrations/
-COPY src ./src/
-COPY tests ./tests/
-COPY views ./views/
-
-RUN yarn install 
-
+COPY . .
```

Comment:
I would rather be more explicit about what goes into that image and keep all the COPY steps :)